common/app/templates/headerInlineJS/loadFonts.scala.js

```diff
@@ -1,87 +1,165 @@
 @()
 
-// Duplicated in /static/src/javascripts/projects/common/utils/detect.js
-// Determine what type of font-hinting we want.
-
 (function (window, document) {
-    if (!window.guardian.isModernBrowser) {
-        return false;
-    }
-    var fontHinting = function () {
-        var ua = navigator.userAgent,
-            windowsNT = /Windows NT (\d\.\d+)/.exec(ua),
-            hinting = 'Off',
-            version;
-
-        if (windowsNT) {
-            version = parseFloat(windowsNT[1], 10);
-
-            // For Windows XP-7
-            if (version >= 5.1 && version <= 6.1) {
-                if (/Chrome/.exec(ua) && version < 6.0) {
-                    // Chrome on windows XP wants auto-hinting
-                    hinting = 'Auto';
-                } else {
-                    // All others use cleartype
-                    hinting = 'Cleartype';
-                }
-            }
-        }
-        return hinting;
-    }();
+    var ua = navigator.userAgent;
 
-    // Check to see if you should get webfonts, and then try to load them from localStorage if so
-    var cookieData = 'GU_fonts=off; domain=' + location.hostname + '; path=/';
-
-    function disableFonts() {
-        document.cookie = cookieData + '; max-age=' + (60 * 60 * 24 * 365);
-    }
+    // Determine what type of font-hinting we want.
+    var fontHinting = (function () {
+        var hinting;
+        function determineHinting() {
+            if (typeof hinting === 'undefined') {
+                try {
+                    var windowsNT = /Windows NT (\d\.\d+)/.exec(ua),
+                        version;
 
-    function enableFonts() {
-        document.cookie = cookieData + '; expires=Thu, 01 Jan 1970 00:00:00 GMT';
-    }
+                    hinting = 'Off';
+                    if (windowsNT) {
+                        version = parseFloat(windowsNT[1], 10);
 
-    function fontsEnabled() {
-        return document.cookie.indexOf('GU_fonts=off') === -1;
-    }
+                        // For Windows XP-7
+                        if (version >= 5.1 && version <= 6.1) {
+                            if (/Chrome/.exec(ua) && version < 6.0) {
+                                // Chrome on windows XP wants auto-hinting
+                                hinting = 'Auto';
+                            } else {
+                                // All others use cleartype
+                                hinting = 'Cleartype';
+                            }
+                        }
+                    }
+                } catch (e) {}
```

Comment:
Can we log it for developer sanity? (Why is this thing not working? I have no errors in my console…)